cci_nepal/analysis/wp2_data_scoping/database_sample/nepal_flood_landslide_descriptive.py

```diff
@@ -0,0 +1,1006 @@
+# -*- coding: utf-8 -*-
+# ---
+# jupyter:
+#   jupytext:
+#     cell_metadata_filter: -all
+#     comment_magics: true
+#     text_representation:
+#       extension: .py
+#       format_name: percent
+#       format_version: '1.3'
+#       jupytext_version: 1.11.4
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# %%
+import pandas as pd
+import numpy as np
+import matplotlib.pyplot as plt
+import cci_nepal
+from cci_nepal.getters.data_scoping import get_sample_data as gsd
+from cci_nepal.pipeline.data_scoping import mis_sample_pipeline as msp
+import logging
+import re
+
+# %matplotlib inline
+
+
+# %%
+project_dir = cci_nepal.PROJECT_DIR
+logging.info(project_dir)
+
+# %%
+# to recognize the used characterset
+# #!pip install openpyxl
+
+# %%
+# read sample nepal flood data for 2020
+data_df = gsd.read_excel_file(f"{project_dir}/inputs/data/PDM_ Datasheet.xlsx")
+
+# %%
+# data on nepal's population by district in 2011 -could be explored to gain insight about the population distribution
+# of the affected areas.
+population_df = gsd.read_csv_file(f"{project_dir}/inputs/data/data.csv")
```

Comment:
Looks good. I think we should rename the file to make it more meaningful and also consider a sub folder in inputs-data as the more data we add the more confusing it will get especially if the names are more generic.